racket/collects/raco/raco.rkt

```diff
@@ -38,13 +39,48 @@
   ((executable-yield-handler) result) ; to enable GUI commands
   (exit result))
 
+(define (parse-cmd-output-option cmdline)
+  (let* ([tout-idx (index-of cmdline "--cmd-output")]
+         [cmd-is-not-help (not (and (pair? cmdline)
+                                    (equal? "help" (car cmdline))))])
+    (if (and tout-idx
+             (< tout-idx (sub1 (length cmdline)))
+             cmd-is-not-help)
+        (string->path (list-ref cmdline (add1 tout-idx)))
```

Comment:
This is unsafe. Users might miss the corresponding parameter. E.g., `raco test --cmd-output`. We want a proper error in that case, not a `list-ref` error.

Reply:
Oh, I just saw that there's a check above that prevents this error.

But that doesn't really solve the problem. There should still be a proper error.